common/config/rush/browser-approved-packages.json

```diff
@@ -1,5 +1,10 @@
 // DO NOT ADD COMMENTS IN THIS FILE.  They will be lost when the Rush tool resaves it.
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/approved-packages.schema.json",
-  "packages": []
+  "packages": [
+    {
+      "name": "long",
```

Comment:
Move this to **nonbrowser-approved-packages.json**.  (This repo doesn't have any browser projects.)  After editing you can run `rush update` to normalize the JSON files.